configs/dictionaries/accountbankhols_percentages.yaml

```diff
@@ -0,0 +1,3 @@
+---
+A: 50
```

Comment:
WIP PA to confirm

Reply:
Maxwell has said that 50/50 is a good split.

Reply:
And now Daisy has said a 20/80 split, which Maxwell has agreed to.